test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -288,6 +290,16 @@ private static boolean isOperatorVersionAbove040() {
         }));
     }
 
+    // Removal of the JVM option 'UseCGroupMemoryLimitForHeap' is required with JVM environments >= 10. This option
+    // is supplied by default by the operators. We cannot 'deactive' it using the XX:- counterpart as it is unrecognized.
```

Comment:
@co-jo can you please file an issue for Pravega Operator to remove usage of `UseCGroupMemoryLimitForHeap` for Pravega 0.9+

Reply:
> 
> 
> @co-jo can you please file an issue for Pravega Operator to remove usage of `UseCGroupMemoryLimitForHeap` for Pravega 0.9+

Sure.